hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/ContextTransactionProcessor.java

```diff
@@ -99,13 +99,13 @@ public ContextTransactionProcessor(
         this.hederaEvmContext = requireNonNull(hederaEvmContext);
         this.hevmTransactionFactory = requireNonNull(hevmTransactionFactory);
         this.gasCharging = requireNonNull(customGasCharging);
-        this.hederaOpsDuration = requireNonNull(hederaOpsDuration);
+        this.contractMetrics = requireNonNull(contractMetrics);
     }
 
     @Override
     public CallOutcome call() {
-        // Apply the latest ops duration schedule from the configuration
-        setOpsDurationValues();
```

Comment:
If we remove this, then the ops duration configuration will not be updated with a 121 file update